doc-content/enterprise-only/admin-and-config/setup-auto-cleanup-proc.adoc

```diff
@@ -14,13 +14,16 @@ You can set up an automatic cleanup job in {CENTRAL}.
 org.jbpm.executor.commands.LogCleanupCommand
 ----
 
-. To use the parameters, complete the following steps:
-+
-For full parameters list, see xref:clean-log-proc[].
+. To configure parameters, complete the following steps:
 +
+--
 .. Open the *Advanced* tab.
 .. Click *Add Parameter*.
 .. Enter a parameter in the *Key* column and enter a parameter value in the *Value* column.
```

Comment:
Cannot see what the columns look like but maybe a column should be treated as a [field](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#field)
```suggestion
.. In the *Key* column, enter a parameter.
.. In the *Value* column, enter a parameter.
```